api/resources/translations/messages-ne.properties

```diff
@@ -1264,7 +1264,10 @@ training_cards.confirm.title = तालिम छोड्ने हो?
 training_cards.error.loading = तालिम लोड गर्दा त्रुटि भयो। तपाइँको सुपरभाइजरलाई सम्पर्क गर्नुहोस्।
 training_cards.error.save = तालिम सेभ गर्न त्रुटि।
 training_cards.form.saved = तालिम सम्पन्न भयो।
-training_cards.modal.title = महत्वपुर्ण परिवर्तनहरु
+training_materials.page.no_more_trainings = अब कुनै तालिम छैन
+training_materials.page.no_selected = कुनै तालिम सामग्री छनोट गरिएको छैन।
+training_materials.page.no_trainings = कुनै तालिम फेला परेन
+training_materials.page.title = प्रशिक्षण सामग्री
```

Comment:
@sugat009 is this good Nepali translation?

Reply:
a bit of change
"no more trainings" = "थप तालिम छैन"
"no_trainings" = "तालिम फेला परेन"
"title" = "तालिम शीर्षक"

Reply:
Thanks Sugat!